src/util.cpp

```diff
@@ -927,6 +922,40 @@ std::string ArgsManager::GetChainName() const
     return CBaseChainParams::MAIN;
 }
 
+std::string ArgsManager::TrimString(const std::string& str, const std::string& pattern)
```

Comment:
This can just be a static function; it doesn't need to be a method of `ArgsManager`.